plugin/plugins/bilibili_danmaku/danmaku_core.py

```
@@ -648,42 +714,67 @@ async def _connect_once(self):
-                            self._process_packet(message)
-                        # str 消息忽略
-                    except Exception as e:
-                        self._log(f"处理消息异常: {e}", "debug")
+            try:
+                async with websockets.connect(ws_url, ping_interval=None, **_ws_kwargs) as ws:
+                    self._ws = ws
+
+                    # 发送认证包
+                    self._connection_state = ConnectionState.AUTHENTICATING
+                    await ws.send(_pack(OPERATION_AUTH, auth_body))
+                    self._log("认证包已发送，等待服务器回复...")
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)